packages/graphql-elasticsearch-transformer/src/resources.ts

```diff
@@ -117,7 +131,6 @@ export class ResourceFactory {
           'c5.4xlarge.elasticsearch',
           't4g.medium.elasticsearch',
           'c6g.4xlarge.elasticsearch',
-          'c6g.xlarge.elasticsearch',
```

Comment:
I believe that the instance type is still supported on few regions. Is there a reason for removing this type?

Reply:
I think the right thing to do long term would be to create per-region mappings for all of our resources. in the meantime, I think would should err on the side of _never showing an invalid option_ as opposed to always showing every available option. See linked issue for more details https://github.com/aws-amplify/amplify-cli/issues/8572